### PR TITLE
Zei/version in status/2172

### PR DESCRIPTION
**Pull request by vinay-satish**
### Description

Added chef-server version in status endpoint

### Issues Resolved

resolves #2147

### Check List

- [ ] All buildkite tests pass

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -346,4 +346,30 @@ To create a separate solr vm create a `config.yml` file with the following conte
...
+
+### [chef/chef-server:master] omnibus/adhoc
+omnibus/adhoc pipeline runs the integration tests on different builds. Integration test covers the API endpoints of the project.
+The pipeline creates different build for different supported OS/environment and pushes the builds to jfrog artifactory.
+The integration tests scripts are tested against each of the different builds created.
+This is pipeline is automatically triggered every night to make sure that the master is always ready to ship.
+
+### [chef/umbrella:master] chef-server
+This pipeline is for end to end testing and creates different builds integrating with other projects of chef.
+These builds are packaged to replicate the different environment in which chef-server will be used by the customers.
```
Comment (PrajaktaPurohit) [inline dev/README.md:374]: Might also be helpful to include: "This pipeline is run on a nightly basis using the latest build from the current omnibus pipeline."

File: dev/README.md [hunk 2]
```diff
@@ -346,4 +346,30 @@ To create a separate solr vm create a `config.yml` file with the following conte
   external_solr:
     start: true
 ```
+# Buildkite
+
```
Comment (PrajaktaPurohit) [inline dev/README.md:350]: Thank you for including this information! I think a better place to put this information might be in the chef-server/docs/HOW_THINGS_WORK.md.

File: dev/README.md (hunk 1 above)
Comment (vinay-satish) [reply dev/README.md:374]: sure will add it

File: dev/README.md (hunk 2 above)
Comment (vinay-satish) [reply dev/README.md:350]: will make the changes

**Review by PrajaktaPurohit — APPROVED** [review]

**Pull request merged**